friends/src/components/Friend.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+import {Link} from 'react-router-dom';
+
+const StyledFriend = styled.div`
+  margin: 5px auto;
+  max-width: 50%;
+  background-color: #2b2b2e;
+  color: white;
+  border: 1px solid black;
+  border-radius: 4px;
+  text-align: left;
+
+  h2,
+  h3 {
+    padding-right: 20px;
+  }
+`;
+
+const UpdateButton = styled.button`
+background-color: #bbbbbb;
+color: black;
+padding: 10px;
+border-radius: 3px;
+`;
+
+const Friend = props => {
+  //console.log(props);
+  //let friend;
+  //console.log(props.friends.find(f => `${f.id}` === props.match.params.id))
+  //if (props.edit) {
+    //friend = props.friends.find(f => `${f.id}` === props.match.params.id);
+  //} else {
+    //friend = props.friend;
+  //}
+  const friend = props.friend
```

Comment:
You could use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here to make this a little cleaner:
```javascript
const { friend } = props;
// and, to get its properties without too much confusion:
const { name, age, email, id } = friend;
```
or even:
```javascript
const { name, age, email, id } = props.friend;
```

Reply:
With the properties of friend being destructured, your return would look more like this:
```javascript
return (
  <StyledFriend>
    <div>
      <h2>{name}</h2>
    </div>
    <div>
      <h3>{age} years old</h3>
    </div>
    <div>
      <h3>{email}</h3>
    </div>
    <Link to={`/update/${id}`}>
      <UpdateButton>Update</UpdateButton>
    </Link>
    <UpdateButton onClick={() => props.deleteFriend(id)}>
      Unfriend
    </UpdateButton>
  </StyledFriend>
);
```